test/cases/enum.ts

```diff
@@ -6,7 +6,8 @@ export var schema = {
       "enum": ["a", "b", "c"]
     },
     "bar": {
-      "enum": [1, 2, 3]
+      "enum": [1, 2, 3],
+      "tsEnumNames": ["One","Two","Three"]
```

Comment:
It's really quite straightforward but I'm not on my work computer right now so I don't have it on hand.  I'll try to recreate from memory:

MediaType.json
````
{
type: "integer",
enum: [1, 2, 3],
tsEnumNames: ["Voice", "Chat", "Email"],
javaEnumNames: ["Voice", "Chat", "Email"]
}
````

It's valid input to jsonschema2pojo and generates me:

````
public enum MediaType {
    Voice(1),
    Chat(2),
    Email(3)

   // various constructor and parsing methods:  java provides values() to get a MediaType[] containing all of them; the generator provides fromValue(int): MediaType and value(): int
   // this is not expressible in TS as enums are just integers and not objects, hence the need for the util class
}
````

Output TS that's needed:
````
export enum MediaType {
    Voice = 1,
    Chat = 2,
    Email = 3
}
// plus the util methods
````

The trick here is I need to specify the enum values *as integers* so that they are transmissible across the json boundary, since the integers are what will be written to and parsed from the json.  Hence the symbolic names won't occur in the json, and therefore aren't valid in `enum` in the schema.